yarn.lock

```diff
@@ -5282,6 +5282,37 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@grafana/data@npm:9.3.0-84974pre, @grafana/data@npm:canary":
```

Comment:
Experimental is pulling in canary versions of grafana packages. I'm not sure we want that. Maybe experimental should set these as peerDependencies so it will use the versions in core? 🤔

Reply:
Good catch! Should be fixed now.